fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -1325,17 +1328,20 @@ void applyMetadataMutations(SpanContext const& spanContext,
                             const VectorRef<MutationRef>& mutations,
                             LogPushData* toCommit,
                             const std::unordered_map<EncryptCipherDomainId, Reference<BlobCipherKey>>* pCipherKeys,
+                            EncryptionAtRestMode encryptMode,
                             bool& confChange,
                             Version version,
                             Version popVersion,
                             bool initialCommit) {
+	TraceEvent("ApplyMetadataMutationsEncryptMode").detail("Mode", encryptMode.toString());
```

Comment:
@sfc-gh-nwijetunga do we intend to log this for every mutation update? Or was it intended as a debug-only thing during development.

Reply:
I'll make it debug only trace in a separate pr

Reply:
Great! However, even with debug-only it might be quite noisy, biggest question to ask here is: `do we see value in having this Trace line ?`

Reply:
Hmm i guess we could remove it, this value comes from the CP and the CP encrypt mode is logged so i guess this would be redundant

Reply:
Yeah, that is my understanding as well. In such case, let's remove it and make sure CP and/or Resolve core TraceEvent (at the time of instantiation) has the required information to assist us with necessary debugging. 

Thanks